### PR TITLE
Increase lira deployment replicas

### PR DESCRIPTION
**Pull request by samanehsan**
Increase the number lira deployment replicas from 1 to 3.

### CODE REVIEW

File: kubernetes/listener-deployment.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: Deployment
 metadata:
   name: listener
 spec:
-  replicas: 1
+  replicas: 3
```
Comment (dshiga) [inline kubernetes/listener-deployment.yaml]: Now that we have the ctmpl file, I wonder if there is any reason to keep this yaml file. I don't think we actually use it for anything, do we?

Comment (samanehsan) [reply kubernetes/listener-deployment.yaml]: That's a good point! I'll go ahead and remove it

**Review by dshiga — APPROVED** [review]
Can you rerun the mintegration test? I'm assuming it failed because caas was down.

**Comment by samanehsan** [comment]
retest this please

**Comment by samanehsan** [comment]
retest this please

**Comment by samanehsan** [comment]
retest this please

**Pull request merged**